Invalid Plugin

Set up multiplatform on my computer, but it took more steps than those in the readme. There was an invalid plugin error after running `ionic serve -c`. I added a new plugin, which made `ionic serve -c` run as expected. Not sure what caused this problem or if the fix was appropriate.

I checked out to master to recreate the error, but that branch doesn't produce it. So, I can't provide the exact error. Maybe I made a mistake earlier on, so here's what happened. 

I setup multiplatform. There was one error popping up after `ionic serve -c:`
```
[ERROR] Error occurred while loading plugins. CLI functionality may be limited.
        Checking for CLI updates now...
[ERROR] No updates found after plugin error--please report this issue.
``` 
(Based off of my search history from last night)

At this point, I looked back at the readme... I hadn't read the "Testing on Devices" section. This is totally my fault. So, I updated the software of my computer. Don't know if that should have fixed anything. However, when I ran `ionic serve -c`, the result was a different error message:

 `[ERROR] PLUGIN INVALID`. 

Can't find the exact message; that's what I copied and pasted into Google. The change I made in the branch I'm working on is below, along with the commands:
```
npm install --save-dev --save-exact @ionic/cli-plugin-ionic-angular@latest
npm install --save-dev --save-exact @ionic/cli-plugin-cordova@latest
```
The diff of `package.json`:
```
-    "@ionic/cli-plugin-cordova": "0.0.9",
-    "@ionic/cli-plugin-ionic-angular": "1.0.0",
+    "@ionic/cli-plugin-cordova": "1.4.0",
+    "@ionic/cli-plugin-ionic-angular": "1.3.1",

``` 
So... I updated the plugins and made them angular-specific? I have a feeling this is my fault, but just in case it isn't, I put the issue up here.

--- a/package.json
+++ b/package.json
@@ -48,10 +48,10 @@
     "zone.js": "0.7.2"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "1.1.4",
+    "@ionic/app-scripts": "^1.3.7",
     "@ionic/cli-build-ionic-angular": "0.0.3",
     "@ionic/cli-plugin-cordova": "0.0.9",
-    "@ionic/cli-plugin-ionic-angular": "1.0.0",
+    "@ionic/cli-plugin-ionic-angular": "^1.3.1",
     "@types/jasmine": "^2.5.41",
     "@types/node": "^7.0.8",
     "angular2-template-loader": "^0.6.2",

--- a/package.json
+++ b/package.json
@@ -50,7 +50,7 @@
   "devDependencies": {
     "@ionic/app-scripts": "^1.3.7",
     "@ionic/cli-build-ionic-angular": "0.0.3",
-    "@ionic/cli-plugin-cordova": "0.0.9",
+    "@ionic/cli-plugin-cordova": "1.4.0",
     "@ionic/cli-plugin-ionic-angular": "^1.3.1",
     "@types/jasmine": "^2.5.41",
     "@types/node": "^7.0.8",

--- a/package.json
+++ b/package.json
@@ -49,8 +49,8 @@
   },
   "devDependencies": {
     "@ionic/app-scripts": "^1.3.7",
-    "@ionic/cli-build-ionic-angular": "0.0.3",
-    "@ionic/cli-plugin-cordova": "1.4.0",
+    "@ionic/cli-build-ionic-angular": "^0.0.3",
+    "@ionic/cli-plugin-cordova": "^1.4.0",
     "@ionic/cli-plugin-ionic-angular": "^1.3.1",
     "@types/jasmine": "^2.5.41",
     "@types/node": "^7.0.8",

